stdlib/os/testdata/test.py

```diff
@@ -181,6 +182,7 @@
         print("INVALID error caught: %s" % e)
     os.remove(fname)
     os.rmdir(dir2)
+    print(os.listdir(top))
```

Comment:
please add a test where `top` is of type `bytes-like`.

Reply:
also, please add a test where the argument of `listdir` is `None` (after having `chdir`ed to _e.g._ `top`)

Reply:
Fixed.